src/FakeItEasy/Core/FakeWrapperConfigurator.cs

```diff
@@ -38,9 +38,9 @@ public override IFakeOptionsForWrappers<T> Wrapping(T wrappedInstance)
             return this.fakeOptions.Wrapping(wrappedInstance);
         }
 
-        public override IFakeOptions<T> WithAdditionalAttributes(IEnumerable<CustomAttributeBuilder> customAttributeBuilders)
+        public override IFakeOptions<T> WithAdditionalAttributes(params Expression<Func<Attribute>>[] customAttributeExpressions)

```

Comment:
"factories" seems to imply we're going to invoke them, which we're not, so it's probably misleading.